metadata-ingestion/src/datahub/cli/ingest_cli.py

```diff
@@ -198,6 +210,116 @@ async def run_func_check_upgrade(pipeline: Pipeline) -> None:
     loop.run_until_complete(run_func_check_upgrade(pipeline))
 
 
+@ingest.command()
+@click.pass_context
+@upgrade.check_upgrade
+@telemetry.with_telemetry()
+@click.option(
+    "-n",
+    "--name",
+    type=str,
+    help="Recipe Name",
+    required=True,
+)
+@click.option(
+    "-c",
+    "--config",
+    type=click.Path(dir_okay=False),
+    help="Config file in .toml or .yaml format.",
+    required=True,
+)
+@click.option(
+    "--executor-id",
+    type=str,
+    default="default",
+    help="Executor id to route execution requests to. Do not use this unless you have configured a custom executor.",
+)
+@click.option(
+    "--cli-version",
+    type=str,
+    help="Provide a custom CLI version to use for ingestion. By default will use server default.",
+    required=False,
+)
+@click.option(
+    "--schedule",
+    type=str,
+    help="Cron definition for schedule. If none is provided, ingestion recipe will not be scheduled",
+    required=False,
+)
+@click.option(
+    "--time-zone",
+    type=str,
+    help=f"Timezone for the schedule. By default uses the timezone of the current system: {tzlocal.get_localzone_name()}.",
+    required=False,
+)
+def deploy(
+    ctx: click.Context,
+    name: str,
+    config: str,
+    executor_id: str = None,
+    cli_version: str = None,
+    schedule: str = None,
+    time_zone: str = tzlocal.get_localzone_name(),
+) -> None:
+    """Deploy ingestion recipe to your DataHub instance."""
+
+    pipeline_config = load_config_file(
+        config,
+        squirrel_original_config=True,
+        squirrel_field="__raw_config",
+        allow_stdin=True,
+        resolve_env_vars=False,
+    )
+
+    pipeline_config.pop("__raw_config")
```

Comment:
this field shouldn't be in there if `squirrel_original_config` is false, which is the default